src/renderer/views/login/MiningView.tsx

```diff
@@ -23,6 +23,7 @@ const MiningView = observer(
       if (setSucceed) {
         standaloneStore.setPrivateKeyEnded(true);
         accountStore.setMiningConfigStatus(true);
+        ipcRenderer.send("set mining");
```

Comment:
Is this change intended? it doesn't seem to be related to MixPanel.

Reply:
It is intended. see also. 
https://github.com/planetarium/9c-launcher/pull/875/files#diff-7adaa52a2eacb085ba5e07e6683a61fb3601547dc00b51fd50fbaf685e13fe96R544-R547

Reply:
🆗